spec/spec_helper.rb

```diff
@@ -28,29 +31,44 @@
   # you configure your source control system to ignore this file.
   config.example_status_persistence_file_path = 'spec/examples.txt'
 
-  DATABASE_NAME = 'delayed_job_chainable_hooks_test'.freeze
```

Comment:
It doesn't look like this gem actually depends on Postgres. Should we switch to sqlite for simpler test setup? [goldiloader](https://github.com/salsify/goldiloader) uses it for tests.

Reply:
After I had so much fun configuring Postgres for CircleCI...

I'll give sqlite a try, makes sense to simplify if it's not too hard to set up.